packages/financial-templates-lib/src/price-feed/utils.js

```diff
@@ -199,6 +199,7 @@ exports.BlockFinder = (requestBlock, blocks = []) => {
   async function findBlock(startBlock, endBlock, timestamp) {
     // In the case of equality, the endBlock is expected to be passed as the one whose timestamp === the requested
     // timestamp.
+    timestamp = Number(timestamp);
```

Comment:
Do we need this cast if it's already casted in getBlockForTimestamp? I don't think this method is externally callable (if I remember correctly).

Reply:
Good point. I had added originally for completeness but will minimize the size of the PR and remove